vpr/src/base/stats.cpp

```diff
@@ -342,7 +342,7 @@ void print_wirelen_prob_dist() {
     int prob_dist_size, i, incr;
 
     prob_dist_size = device_ctx.grid.width() + device_ctx.grid.height() + 10;
-    prob_dist = (float*)vtr::calloc(prob_dist_size, sizeof(float));
+    prob_dist = new float[prob_dist_size]();
```

Comment:
Again, does the standard guarantee 0 initialization of all elements?  If not, write a loop to set to 0.  Even if it does, comment it.